R/tokenization.R

```diff
@@ -82,9 +84,12 @@ wordpiece_tokenize <- function(text,
       max_chars = max_chars
     )
   )
-  ids <- vocab[token_vector]
+  # Get IDs by position. Note that this means that the explicit integer
+  # values in the original vocab don't mean anything anymore. We should
+  # probably do something to address this.
```

Comment:
I *think* my vote would be to expect character vector for the vocab, and then explain in help that it should be in order by index, and that it will be 0-indexed for consistency with Python.

And then make some sort of validate_vocab function that's s3. If it's an integer vector, convert it, explicitly checking that the indexing works, and maybe explicitly throw a warning about character vocabs being safer?